public/script.js

```diff
@@ -1,7 +1,7 @@
-import { drawTilemap, helpGetPlatforms, levelSelect } from "./js/Tilemap.js";
```

Comment:
changed code to use the direct variables that are exported instead of the helper functions I had written.